src/traces/treemap/draw_descendants.js

```diff
@@ -185,14 +194,6 @@ module.exports = function drawDescendants(gd, cd, entry, slices, opts) {
             isHeader: isHeader
         });
 
-        if(helpers.isOutsideText(trace, pt)) {
-            // consider in/out diff font sizes
-            pt.transform.targetY -= (
-                helpers.getOutsideTextFontKey('size', trace, pt, fullLayout.font) -
-                helpers.getInsideTextFontKey('size', trace, pt, fullLayout.font)
-            );
-        }
-
```

Comment:
Why did you :hocho: this?

Reply:
With that in place the text was printed outside the domain on this mock: https://github.com/plotly/plotly.js/blob/429324d66493681f67751fff07503dba243bd332/test/image/baselines/treemap_values.png

Reply:
Great. Nice catch. Resolved.